runtime/ggma/examples/generate_text/tinyllama/tinyllama.pipeline

```diff
@@ -0,0 +1,10 @@
+decode: |
+  fuse.attention.py < decode_.circle
+      | reshape.io.py input --by_shape [1,16,30,4] [1,16,32,4]
+      | transpose.io.kvcache.py > decode.circle
+
+merge: |
+  merge.circles.py prefill.circle decode.circle
+      | fuse.bmm_lhs_const.py
+      | downcast.input_ids.py
```

Comment:
I will use `int32` instead of `int64` (← the default type from `TICO` generated) for input_ids, which is given by `gather`.